### PR TITLE
fix LiquidCrystal_I2C url

### PR DESCRIPTION
**Pull request by cbcGirard**
Original repo URL (https://github.com/marcoschwartz/LiquidCrystal_I2C) now redirects to https://github.com/johnrickman/LiquidCrystal_I2C, preventing installation through library manager. Updated to redirected URL.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @cbcGirard. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/05/13 14:15:21 Loaded 6904 libraries from DB
Changing URL of library LiquidCrystal I2C from https://github.com/marcoschwartz/LiquidCrystal_I2C.git to https://github.com/johnrickman/LiquidCrystal_I2C.git
Success!
```

**Pull request merged**
